content/1. Leertaken/WEBDEV/Security/Workshops/10. Workshop Git/2. Git Repositories.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 2. Git Repositories
+taxonomie: None
+tags: 
+draft: true
+---
+
+> [!warning] Al een beetje bekend met Git? Dan kun je het eerste deel van deze workshop overslaan en doorgaan vanaf [[3. Git Forks & Branches]]
+
+## Opdracht 1: Repository aanmaken
+Een repository is de opslagplaats van je project. Hierin staan alle bestanden en de volledige geschiedenis van hun wijzigingen.
+
+**Stap 1:** Log in op GitHub en ga naar de homepagina of je dashboard.
+
+**Stap 2:** Klik op de groene ‘New’ knop aan de linkerkant van de pagina.
+
+> [!info] Houd de naam van je repository simpel en beschrijvend. Vermijd spaties en speciale tekens (behalve -).
+
+**Stap 2.1:** Geef de repository een naam (bijvoorbeeld FirstRepository) en een beschrijving. Kies ook of je de repository public of private wilt maken.
+
+**Stap 2.2:** Voeg een .gitignore bestand toe met het template voor VisualStudio.
```

Comment:
Zou het niet leerzamer zijn als ze beginnen zonder .gitignore? Dan zullen ze later ontdekken dat ze er een nodig zijn, en dan kun je ze wel de standaard vs template gitignore wel geven.

Reply:
Cool. Ik kan dat hier wel uit halen en een 4e opdracht toevoegen voor .gitignore. Ze zien dan bij opdracht 3 dat je al die files anders in je commit krijgt.